dvc/stage.py

```diff
@@ -731,7 +731,7 @@ def check_can_commit(self, force):
             msg += " and " if (changed_deps and changed_outs) else ""
             msg += "outputs {}".format(changed_outs) if changed_outs else ""
             msg += "md5" if not (changed_deps or changed_outs) else ""
-            msg += " of '{}' changed. Are you sure you commit it?".format(
+            msg += " of '{}' changed. Are you sure you want to commit it?".format(
```

Comment:
The style check has failed https://travis-ci.com/iterative/dvc/jobs/242359132#L243 . This line is too long. Have you installed pre-commit hooks as advised in our [contribution guide](https://dvc.org/doc/user-guide/contributing)?

Reply:
This commit was done by the github web IDE (check the branch name "patch-1"), so I did not have access to git hooks. I did check the `.pylintrc` file and saw that it should be fine with line length (it is 100 there), but I did not open the `.flake8` (which is the one used in your hook).

Commit updated & opened #2578 to bring the two lint files' line length setting to parity.